### PR TITLE
Storing founders info for companies from ANTAC data

### PR DESCRIPTION
**Pull request by IegorSoboliev**

### CODE REVIEW

File: business_register/converter/additional_uo.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
+from data_ocean.converter import Converter
+from business_register.converter.business_converter import BusinessConverter
+from business_register.models.company_models import Company, FounderFull
+
+class FoundersAdder(Converter, BusinessConverter):
```
Comment (criticus) [inline business_register/converter/additional_uo.py:5]: The adder is a relatively small, stocky snake that prefers woodland, heathland and moorland habitats 🐍

File: business_register/converter/additional_uo.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    
+    def __init__(self):
+        self.all_companies_dict = self.put_all_objects_to_dict('name', 'business_register', 
+                                                                "Company")
+        
+        super().__init__()
+
+    def add_founders_info(self, record):
+        founder_name = record.xpath('FOUNDER_NAME')[0].text
+        if founder_name:
```
Comment (criticus) [inline business_register/converter/additional_uo.py:15]: consider making early returns

File: business_register/models/company_models.py [hunk 3]
```diff
@@ -75,6 +75,9 @@ class ExchangeDataCompany(DataOceanModel):
 class FounderFull(DataOceanModel):
     company = models.ForeignKey(Company, on_delete=models.CASCADE, related_name='founders')
     name = models.TextField(null=True)
+    name2 = models.CharField(max_length=100, null=True)
```
Comment (criticus) [inline business_register/models/company_models.py:78]: what is `name2`?

Comment (roman-tiukh) [reply business_register/models/company_models.py:78]: need a meaningful field name

File: business_register/converter/additional_uo.py (hunk 1 above)
Comment (IegorSoboliev) [reply business_register/converter/additional_uo.py]: renamed

File: business_register/converter/additional_uo.py (hunk 2 above)
Comment (IegorSoboliev) [reply business_register/converter/additional_uo.py]: done

File: business_register/models/company_models.py (hunk 3 above)
Comment (IegorSoboliev) [reply business_register/models/company_models.py]: in the end made a separate Model

**Pull request merged**